TransferMethod/Sources/Coordinators/TransferMethodCoordinators.swift

```diff
@@ -21,6 +21,7 @@ import Common
 #endif
 import UIKit
 
+/// SelectTransferMethodTypeCoordinator
```

Comment:
Update to - /// Coordinator class for SelectTransferMethodType

Reply:
Updated